### PR TITLE
[Magiclysm] attunements stuff

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Describe the solution
Added description to all enchantments for attunements
fix soulfire's soul_afterburner spell to be useful
Some tiny changes here and there
#### Testing
Looks okay

### CODE REVIEW

File: data/mods/Magiclysm/enchantments/Blood_Mage.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 [
   {
     "type": "enchantment",
-    "id": "BLOOD_MAGE_HEALING",
+    "id": "BLOOD_MAGE",
+    "name": { "str": "Ichor" },
+    "description": "Your Blood Mage abilities grant you ability to steal some health from your enemies time to time, and constant immune to bleed.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchantments/Blood_Mage.json:6]:
```suggestion
    "description": "Your Blood Mage abilities grant you the ability to steal some health from your enemies from time to time, and immunity to bleeding.",
```

**Review by KorGgenT — COMMENTED** [review]
some grammar touchups and some minor rewording suggestions

File: data/mods/Magiclysm/enchantments/Permafrost_Mage.json [hunk 2]
```diff
@@ -9,14 +9,16 @@
...
-    "max_damage": -125,
+    "max_damage": -225,
     "min_range": 2,
     "max_range": 2
   },
   {
     "type": "enchantment",
     "id": "PERMAFROST_MAGE",
     "condition": "ALWAYS",
+    "name": { "str": "Iceberg" },
```
Comment (GuardianDll) [inline data/mods/Magiclysm/enchantments/Permafrost_Mage.json:20]:
```suggestion
    "name": { "str": "Blizzard" },
```

File: data/mods/Magiclysm/enchantments/Void_Mage.json [hunk 3]
```diff
@@ -32,26 +34,11 @@
...
-    "opposite_part": "dark_eye",
-    "side": "both",
-    "connected_to": "head",
-    "accusative": { "ctxt": "bodypart_accusative", "str": "dark eye" },
-    "hp_bar_ui_text": "dark eye",
-    "base_hp": 20
+    "type": "effect_type",
+    "id": "void_mage_night_vision",
+    "name": [ "" ],
+    "desc": [ "" ],
```
[the remark this answers is not in the document]
Comment (GuardianDll) [reply data/mods/Magiclysm/enchantments/Void_Mage.json:40]: It is intended, info about the effect in the enchantment

**Pull request merged**